### PR TITLE
ddl, util: Add highlight to log

### PR DESCRIPTION
**Pull request by zimulala**
I only add highlight log to highlight type.
`logrus` default color only supports "[loglevel]" and "file=xxx.go" fields to have colors.
I think using `highlight` to debug more easily. In order not to affect everyone's use, I add a new type.

### CODE REVIEW

File: util/logutil/log.go [hunk 1]
```diff
@@ -110,9 +110,30 @@ func stringToLogLevel(level string) log.Level {
 	return defaultLogLevel
 }
 
+// logTypeToColor converts the Level to a color string.
+func logTypeToColor(level log.Level) string {
```
Comment (shenli) [inline util/logutil/log.go]: How about change this to `func logTypeToColor(type string, level log.Level) string `
If type is not "highlight", return "\033".

Comment (coocood) [reply util/logutil/log.go]: Only 'highlight` need to call this function and write color code.

**Comment by coocood** [comment]
LGTM

File: util/logutil/log.go (hunk 1 above)
Comment (shenli) [reply util/logutil/log.go]: So why not call it logLevelToColor?

File: util/logutil/log.go [hunk 2]
```diff
@@ -123,6 +144,12 @@ func (f *textFormatter) Format(entry *log.Entry) ([]byte, error) {
 	} else {
 		b = &bytes.Buffer{}
 	}
+
+	if !f.DisableColors {
```
Comment (shenli) [inline util/logutil/log.go]: Rename DisableColors to EnableColors. It will be more clear.

**Pull request merged**